docs/DebuggingGuide.md

```diff
@@ -21,7 +21,7 @@ Additional setup guidance can be found in the [Contributing.md](https://github.c
 
     `git clone https://github.com/YOURGITHUBACCOUNT/sarif-pattern-matcher`
 
-2. Run BuildAndTest.exe from the root of the enlistment. This will properly load all submodules and validate some machine configuration/settings. Complete any recommended installs/upgrades (e.g., to install a newer sdk via VS installer individual components):
+2. Run BuildAndTest.cmd from the root of the enlistment. This will properly load all submodules and validate some machine configuration/settings. Complete any recommended installs/upgrades (e.g., to install a newer sdk via VS installer individual components):
```

Comment:
>BuildAndTest [](http://example.com/codeflow?start=7&length=12)

Why are we building twice? i.e., once in #2 and again in #0? Maybe we can just indicate that the user should build again in the IDE.